Source/Libs/GLibSharp/HookList.cs

```diff
@@ -12,58 +12,58 @@ namespace GLib {
 	public partial class HookList : GLib.Opaque {
 
 		// Internal representation of the wrapped ABI structure.
-		static public AbiStruct abi_info = new AbiStruct(new List<AbiField> {
+		static public unsafe AbiStruct abi_info = new AbiStruct(new List<AbiField> {
 				new GLib.AbiField("seq_id"
 						, 0
-						, (uint) Marshal.SizeOf(typeof(ulong)) // seq_id
+						, (uint) sizeof (ulong) // seq_id
 						, null
 						, "hook_size"
 						, (long) Marshal.OffsetOf(typeof(GHookList_seq_idAlign), "seq_id")
 						, 0
 						),
 				new GLib.AbiField("hook_size"
 						, -1
-						, (uint) Marshal.SizeOf(typeof(uint)) // hook_size
+						, (uint) sizeof (uint) // hook_size
 						, "seq_id"
 						, "is_setup"
 						, 1
 						, 16
 						),
 				new GLib.AbiField("is_setup"
 						, -1
-						, (uint) Marshal.SizeOf(typeof(bool)) // is_setup
+						, (uint) sizeof (bool) // is_setup
```

Comment:
bool is not blittable type. This should be `sizeof(uint)`, or it should stay as `Marshal.SizeOf`

Reply:
i would prefer `Marshal.SizeOf<bool>()` who knows if it's always uint on all platforms / cpu architectures ...

Reply:
@lytico I'm done that.